.idea/inspectionProfiles/Druid.xml

```diff
@@ -311,6 +311,16 @@
         <constraint name="k" within="" contains="" />
         <constraint name="v" within="" contains="" />
       </searchConfiguration>
+      <searchConfiguration name="Prohibit Thread.getState() != TERMINATED antipattern" text="$t$.getState()!=Thread.State.$state$" recursive="true" caseInsensitive="true" type="JAVA">
+        <constraint name="t" within="" contains="" />
+        <constraint name="state" within="" contains="" />
+        <constraint name="t1" within="" contains="" />
```

Comment:
Did you change these patterns via IntelliJ or directly in plain-text? There is no `t1` variable currently in the pattern. Please add a sample "wrong" code somewhere in the project locally and check within IntelliJ that these pattern work by invoking the structural search.

Reply:
@leventov does this require further changes?